R/plumber.R

```diff
@@ -811,6 +811,10 @@ plumber <- R6Class(
     globalSettings = list(info=list()), # Global settings for this API. Primarily used for Swagger docs.
 
     errorHandler = NULL,
+    setErrorHandlerInternal = function(errorhandler){
```

Comment:
I would delete this method as there is already a public `setErrorHandler` method. (This can be reached using `self$setErrorHandler` in the `evaluateBlock` call above.)

Reply:
Thanks, for this pointer that's exactly what I was missing when trying to access `setErrorHandler` earlier.